components/credentials/src/test/java/org/cloudfoundry/credhub/generators/CertificateGeneratorTest.java

```diff
@@ -264,7 +264,7 @@ public void whenSelfSignIsTrue_itGeneratesAValidSelfSignedCertificate() throws E
       equalTo(CertificateFormatter.pemOf(rootCaKeyPair.getPrivate())));
     assertThat(certificateCredential.getCertificate(),
       equalTo(CertificateFormatter.pemOf(certificate)));
-    assertThat(certificateCredential.getCa(), equalTo(CertificateFormatter.pemOf(certificate)));
+    assertThat(certificateCredential.getCa(), equalTo(""));
```

Comment:
I don't believe this change would be positive overall for Credhub users. It would also be a breaking one.